Formula/voltdb.rb

```diff
@@ -14,13 +14,21 @@ class Voltdb < Formula
   end
 
   depends_on :ant => :build
+  depends_on "cmake" => :build
 
   def install
     system "ant"
 
-    inreplace Dir["bin/*"] - ["bin/voltadmin", "bin/voltdb", "bin/rabbitmqloader"],
+    inreplace Dir["bin/*"] - ["bin/voltadmin", "bin/voltdb", "bin/rabbitmqloader", "bin/voltdeploy", "bin/voltenv"],
```

Comment:
At this point I wonder if it's worth specifying the actual files rather than removing from a wildcard.